Cargo.toml

```diff
@@ -14,3 +14,6 @@ amethyst-imgui = { git = "https://github.com/awpteamoose/amethyst-imgui.git", br
 amethyst-inspector = { git = "https://github.com/awpteamoose/amethyst-inspector.git", branch = "amethyst-v0.10" }
 smart-default = "0.5.2"
 serde = "1.0.90"
+
+[dependencies.shred]
+features = ["nightly"]
```

Comment:
Was this for debugging? If yes, can you revert this? I don't think we should include nightly features

Reply:
Sorry. Forgot to remove it.